app/assets/src/components/common/MetadataUpload.jsx

```diff
@@ -60,6 +61,17 @@ class MetadataUpload extends React.Component {
     this.props.onMetadataChange({ metadata, wasManual: true });
   };
 
+  getCSVUrl = () => {
```

Comment:
I wonder if building these urls should also be part of the `api`.
Maybe on `api/urls.js` ? Wdyt?

Reply:
I think we can hold off for now, as these dynamic `<a>` urls are pretty rare and we already have a re-usable `getURLParamString` function. Maybe in the future.